### PR TITLE
#50 course page responsive

### LINKED ISSUE #50: Make course page responsive
:neckbeard:

### PR DESCRIPTION
**Pull request by JagrajAulakh**
Closes #50 

Makes course page responsive

image

image

### CODE REVIEW

File: src/components/CoursePage/WhatYouWillLearn.js [hunk 1]
```diff
@@ -16,30 +18,35 @@ const WhatYouWillLearn = ({ whatYouWillLearn, enrollBtnURL }) => {
 			<div className="text-white max-w-2xl mx-auto">
 				{whatYouWillLearn.map((info, index) => (
 					<div key={index}>
-						<Accordion>
+						<Accordion className="my-2">
```
Comment (WBawa) [inline src/components/CoursePage/WhatYouWillLearn.js:21]: why is this called my-2? make it a more descriptive name, please.

Comment (JagrajAulakh) [reply src/components/CoursePage/WhatYouWillLearn.js:21]: :neckbeard:

**Review by WBawa — CHANGES_REQUESTED** [review]
please don't use my-2

**Review by WBawa — APPROVED** [review]

File: src/components/CoursePage/WhatYouWillLearn.js (hunk 1 above)
Comment (alialtimimy) [reply src/components/CoursePage/WhatYouWillLearn.js:21]: hello

**Review by alialtimimy — APPROVED** [review]
testing purposes

**Review by zainaraza43 — COMMENTED** [review]
I need my 👑 @hassanuahmad to approve

**Review by zainaraza43 — APPROVED** [review]
good work 👑

**Pull request merged**